OpenBazaar/OpenBazaar-Client
# Blocked users patch

This branch adds functionality to the Blocked Users tab in Settings to add in real-time a blocked user.

to test:
1.) type a message to a user so that they appear in the chat app
2.) navigate to the Blocked User tab of your Settings
3.) from the chat app, block the user
4.) the user should immediately appear on the Blocked User tab***

**\* An exception to that is if, on the off chance, you have more than 25 users
and you haven't scrolled down to lazy load them all. In that case, the user will
not appear instantly, but will be at the bottom of the list once you scroll to the
end.
